nsxt/resource_nsxt_policy_ipsec_vpn_local_endpoint.go

```diff
@@ -354,3 +358,20 @@ func nsxtVPNServiceResourceImporter(d *schema.ResourceData, m interface{}) ([]*s
 
 	return []*schema.ResourceData{d}, nil
 }
+
+func validateIPv6LocaleServiceConflict(ctx context.Context, diff *schema.ResourceDiff, meta interface{}) error {
+	return validateIPv6WithLocaleService(diff, "local_address", "validateIPv6LocaleServiceConflict")
+}
+
+func validateIPv6WithLocaleService(diff *schema.ResourceDiff, fieldName, logPrefix string) error {
+	address := diff.Get(fieldName).(string)
+	servicePath := diff.Get("service_path").(string)
+	ip := net.ParseIP(address)
+	isIPv6 := ip != nil && strings.Contains(address, ":") && ip.To4() == nil
+	isLocaleService := strings.Contains(servicePath, "/locale-services/")
+
+	if isIPv6 && isLocaleService {
+		return fmt.Errorf("IPv6 is not supported with locale-service, refer to documentation")
```

Comment:
"refer to documentation" to users might appear as a reference to Terraform Provider docs, where we don't go into these details (and we shouldn't). Rather than changing to "refer to NSX documentation" I would simply remove "refer to documentation".

However, a more helpful message would be something like:

"IPv6 addresses are not supported for VPN services configured with locale service path. Please use a VPN service configured with a gateway path"